### PR TITLE
[WK-6943]-update-readme

### PR DESCRIPTION
**Pull request by Bliss00**
Added info about downloading models

### CODE REVIEW

**Review by denis-hoz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ static let clientConfig = "" // <-- Change license here
         * zipped size: 3.8 MB
         * unzipped size: 9.3 MB
         * cache for neural networks and auxiliary files: 25 MB
-        * the size of cache for 3D models can be set up via API
+        * the size of cache for 3D models can be set up via API. (3D model size varies depending on it's 'quality. It is downloaded right after session starts.)
```
Comment (nikita-wannafashion) [inline README.md:159]: `'quality`

What this ' means?

Comment (Bliss00) [reply README.md:159]: same as in the next point - _RAM: up to 600 MB, depending on the size and quality of a 3D model_

Comment (Bliss00) [reply README.md:159]: @nikita-wannafashion  what do you suggest for paraphrasing?

Comment (NadyaG) [reply README.md]: I suggest converting this to a Note and removing the period at the end of the list item:
* the size of cache for 3D models can be set up via API<br />**Note:** 3D model size varies depending on its quality. The model will be downloaded right after session starts.
@Bliss00

**Review by denis-hoz — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (NadyaG) [reply README.md]: At least please change it's -> its.

**Review by denis-hoz — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -156,7 +156,8 @@ static let clientConfig = "" // <-- Change license here
         * zipped size: 3.8 MB
         * unzipped size: 9.3 MB
         * cache for neural networks and auxiliary files: 25 MB
-        * the size of cache for 3D models can be set up via API. (3D model size varies depending on it's quality. It is downloaded right after session starts.)
+        * the size of cache for 3D models can be set up via API. 
+        **Note:** 3D model size varies depending on its quality. The model is downloaded right after session starts.
```
[the remark this answers is not in the document]
Comment (denis-hoz) [reply README.md:160]: true https://github.com/WANNABY/WannaSDK-iOS/tree/WK-6943-update-readme

[the remark this answers is not in the document]
Comment (Bliss00) [reply README.md]: Added, now it seems ok.  https://github.com/WANNABY/WannaSDK-iOS/tree/WK-6943-update-readme

**Review by anastasia-sazonova-l — APPROVED** [review]

**Review by denis-hoz — APPROVED** [review]

**Pull request merged**